store/tikv/gcworker/gc_worker.go

```diff
@@ -515,19 +515,51 @@ func doGC(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier st
 	// Sleep to wait for all other tidb instances update their safepoint cache.
 	time.Sleep(gcSafePointCacheInterval)
 
+	log.Infof("[gc worker] %s start gc, safePoint: %v.", identifier, safePoint)
+	startTime := time.Now()
+	regions := 0
+	errRegions := 0
+
+	bo := tikv.NewBackoffer(tikv.GcOneRegionMaxBackoff, goctx.Background())
+	var key []byte
+	for {
+		select {
+		case <-ctx.Done():
+			return errors.New("[gc worker] gc job canceled")
+		default:
+		}
+
+		loc, err := store.GetRegionCache().LocateKey(bo, key)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		if err = doGCForOneRegion(ctx, store, safePoint, identifier, loc.StartKey, loc.EndKey); err != nil {
+			regions++
+		} else {
+			errRegions++
+		}
+
+		key = loc.EndKey
+		if len(key) == 0 {
+			break
+		}
+	}
+	log.Infof("[gc worker] %s finish gc, safePoint: %v, regions: %v, ignored regions: %v, cost time: %s", identifier, safePoint, regions, errRegions, time.Since(startTime))
```

Comment:
I think we can output this line of log at regular intervals (1min for example) during gc, so that we can know the GC progress.

Reply:
ok